### DISCUSSION TITLE
Map visualizations based on aswkt coordinates

### DISCUSSION THREAD

**Discussion by jerfeygilbert** [zazuko/trifid#97] [Q&A] [answered]
Hi, 

We would like to create map visualizations based on aswkt coordinates. Based on the following types ‘Sdo:Polygon / Point’ and ‘Bag:geometrie/vlakGeometrie’. 

What would be the way to do this? Unfortunately we haven’t acquired results yet. Is this solution / discussion the way to go? ("/zazuko/trifid/discussions/86") https://github.com/zazuko/trifid/discussions/86

If so, how can we add the background map? Is it done by this iframeTemplate variable?
 [https://gist.github.com/ktk/1289a05f06a321fdd7619b0a678b43af]
var iframeTemplate = '<iframe src="[https://map.geo.admin.ch/embed.html?${mapId}]" style="width:100%; height: 300px; border: 0;"></iframe>'

Example link of a polygon we would like to visualize:
[https://labs.kadaster.nl/trifid-kg/kadaster/kg/id/geo-shape/bag-0200100000001088]

Thanks in advance!

**Comment by bergos** [comment, marked as answer]
Please have a look at the Basel Trifid instance. It uses Leaflet to render the map. [Here](https://ld.bs.ch/cube/100052/observation/BS014722) is an example resource. Check the last lines of the HTML file and [this](https://ld.bs.ch/js/map.js) JavaScript file. See the [Leaflet docs](https://leafletjs.com/SlavaUkraini/reference.html#tilelayer) how to change the tile layer.

**Reply by jerfeygilbert** [reply]
Thanks for the help and quick response! We successfully created map visualizations based on these pointers and JavaScript.
